content/200-concepts/100-components/01-prisma-schema/08-views.mdx

```diff
@@ -123,7 +123,7 @@ db.createView('UserInfo', 'User', [
 
 ## Use views with Prisma Migrate and <inlineCode>db push</inlineCode>
 
-If you apply changes to your schema with Prisma Migrate or `db push`, Prisma does not create or run any SQL related to views.
+Prisma does not create or run any SQL related to views.
```

Comment:
This is not correct. When you put it there yourself, Prisam Migrate happily runs the custom SQL which can be for views.

(The sentence was slightly incorrect before, now without the mention of "Prisma Migrate or `db push`" this is even more so I would say.)

Reply:
Reverted this to its previous state, but emphasized changes to your "prisma schema" do not work with Migrate or `db push` to help enforce the fact that it is only a limitation in within Prisma schema, not the actual migration file.